test/channels/service/ChannelsService.spec.ts

```diff
@@ -119,6 +120,22 @@ describe('Channels Service', () => {
       enabled: true
     };
 
+    const updateChannel: ChannelUpdateRequest = {
+      additionalFields: [{
+        name: 'inbound_orders',
+        value: 'true'
+      }],
```

Comment:
why do we need to make update request with all the fields which are inchannelCreateRequest when channelUpdateRequest accepts only "credentials"? I am assuming that just need an "Id" and "credentials" to update.

Reply:
@nagarajkovi  actually as per given story we need to update channel with credentials before this we didn't have update call for channel.
So i have referred other service update call based on that i have made this changes.